.github/workflows/bump_version.yaml

```diff
@@ -1,64 +1,65 @@
         name: Get version bump
-        runs-on: ubuntu-latest
-        needs: industrial_ci
-        outputs:
-            bump: ${{ env.BUMP }}
-        steps:
-            - if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
-              name: Get version bump
-              id: get-version-bump
-              uses: husarion-ci/action-get-version-bump@v0.3.0
-            - if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
-              run: echo "BUMP=${{ steps.get-version-bump.outputs.bump }}" >> $GITHUB_ENV
-            - if: github.event_name == 'workflow_dispatch'
-              run: echo "BUMP=${{ github.event.inputs.name }}" >> $GITHUB_ENV
+        id: get-version-bump
+        uses: husarion-ci/action-get-version-bump@v0.3.0
+      - if: github.event_name == 'pull_request' && github.event.pull_request.merged == true
+        run: echo "BUMP=${{ steps.get-version-bump.outputs.bump }}" >> $GITHUB_ENV
+      - if: github.event_name == 'workflow_dispatch'
+        run: echo "BUMP=${{ github.event.inputs.name }}" >> $GITHUB_ENV
 
-    catkin-release:
-        name: Bump version
-        runs-on: ubuntu-latest
-        needs: get-bump
-        outputs:
-            new_version: ${{ steps.catkin-release.outputs.new_version }}
-        steps:
-            - name: Checkout
-              uses: actions/checkout@v2
-            - name: Catkin release
-              id: catkin-release
-              uses: husarion-ci/action-catkin-release@v0.1.4
-              with:
-                  bump: ${{ needs.get-bump.outputs.bump }}
-                  github_token: ${{ secrets.GITHUB_TOKEN }}
-                  git_user: action-bot
-                  git_email: action-bot@action-bot.com
+  catkin-release:
+    name: Bump version
+    runs-on: ubuntu-latest
+    needs: get-bump
+    outputs:
+      new_version: ${{ steps.catkin-release.outputs.new_version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Catkin release
+        id: catkin-release
+        uses: husarion-ci/action-catkin-release@v0.1.4
+        with:
+          bump: ${{ needs.get-bump.outputs.bump }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          git_user: action-bot
+          git_email: action-bot@action-bot.com
 
-    build-and-push-docker-image:
-        name: Create new docker image
-        runs-on: ubuntu-latest
-        needs: catkin-release
-        steps:
-            - name: trigger the endpoint
-              run: >
-                  curl -X POST
-                  -H "Accept: application/vnd.github+json"
-                  -H "Authorization: Bearer ${{ secrets.GH_PAT }}"
-                  https://api.github.com/repos/husarion/rosbot-xl-docker/dispatches
-                  -d '{"event_type":"ros-package-update","client_payload":{"image_version":"${{ needs.catkin-release.outputs.new_version }}"}}'
+  build-and-push-docker-image:
```

Comment:
I would change the name of this step to `Trigger new docker image build on rosbot-docker`

Reply:
Maybe `Trigger rosbot-docker to build image`?
I would like to change this name